helm/oauth2-proxy/values.yaml

```diff
@@ -153,6 +153,9 @@ service:
   externalTrafficPolicy: ""
   # configure internalTrafficPolicy
   internalTrafficPolicy: ""
+  # configure service target port
+  # useful when there's need to route requests to sidecar containers first
```

Comment:
I would remove this comment.

Reply:
fixed in [01147b8](https://github.com/oauth2-proxy/manifests/pull/270/commits/01147b849ac7afbc246a5d20b291a2ff2f8d8134)